14 AsyncCalls_Redux_Saga/src/common/constants/actionTypes.ts

```diff
@@ -5,4 +5,5 @@ export const actionTypes = {
   UPDATE_MEMBER_FIELD: 'UPDATE_MEMBER_FIELD',
   SAVE_MEMBER: 'SAVE_MEMBER',
   FETCH_REPOSITORIES_COMPLETED:'FETCH_REPOSITORIES_COMPLETED',
+  FETCH_MEMBER_REQUEST : 'FETCH_MEMBER_REQUEST',
```

Comment:
There is a FETCH_MEMBERS_REQUEST_SAGA that should be renamed to FETCH_MEMBER_REQUEST_COMPLETED

Reply:
I think we need some clean up here, we have tasks:

We got two FETCH_MEMBERS_COMPLETED (saga and normal)

We got FETCH_MEMBER_BY_ID and REQUEST%..

Just go through them and remove the ones not in use, and rename the ones that are useful (using F2)